R/packrat-mode.R

```diff
@@ -182,6 +182,10 @@ afterPackratModeOn <- function(project,
     options(repos = repos)
   }
 
+  # Set a secure download method if any of the repos URLs use https
```

Comment:
I think the fix is still required (if we don't set the repos however it still has https URLs then we need to set the download method anyway)